### PR TITLE
Black all the things

### PR DESCRIPTION
**Pull request by j-bennet**
Running `black .` shouldn't make unintended changes. We can use `pyproject.toml` to keep `black` rules.

### CODE REVIEW

**Review by pquentin — CHANGES_REQUESTED** [review]
Thanks for fixing this! However, we are now storing the list of included folders in two places. What are your thoughts on just running black/isort on everything, including changelog.py and docs/conf.py?

This would fix `black .`, remove some extra config, and allow me to edit changelog.py and docs/conf.py with my editor without ruining the formatting.

**Comment by j-bennet** [comment]
After running `black` on everything, 3 files ended up with changes:

```
	modified:   changelog.py
	modified:   docs/conf.py
	modified:   docs/create_scheduler_plot.py
```

If there's no specific reason to include `docs`, I'd rather `black` everything.

File: docs/conf.py [hunk 1]
```diff
@@ -30,17 +30,15 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-extensions = [
-    "sphinx.ext.ifconfig"
-]
+extensions = ["sphinx.ext.ifconfig"]
```
Comment (pquentin) [inline docs/conf.py:33]:
```suggestion
extensions = ["sphinx.ext.ifconfig",]
```

File: docs/conf.py [hunk 2]
```diff
@@ -51,14 +49,10 @@ def read_min_python_version():
...
-        raise ConfigError(
-            f"Failed building docs as required key [{e}] couldn't be found in the file [{CI_VARS}]."
-        )
+        raise ConfigError(f"Failed building docs as required key [{e}] couldn't be found in the file [{CI_VARS}].")
 
 
-GLOBAL_SUBSTITUTIONS = {
-    "{MIN_PY_VER}": read_min_python_version()
-}
+GLOBAL_SUBSTITUTIONS = {"{MIN_PY_VER}": read_min_python_version()}
```
Comment (pquentin) [inline docs/conf.py:55]:
```suggestion
GLOBAL_SUBSTITUTIONS = {"{MIN_PY_VER}": read_min_python_version(),}
```

File: docs/conf.py [hunk 3]
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
...
+htmlhelp_basename = "Rallydoc"
 
 # -- Options for LaTeX output ---------------------------------------------
 
 latex_elements = {}
 
 latex_documents = [
-    (master_doc, 'Rally.tex', 'Rally Documentation',
-     'Daniel Mitterdorfer', 'manual'),
+    (master_doc, "Rally.tex", "Rally Documentation", "Daniel Mitterdorfer", "manual"),
```
Comment (pquentin) [inline docs/conf.py:167]:
```suggestion
    (master_doc, "Rally.tex", "Rally Documentation", "Daniel Mitterdorfer", "manual",),
```

File: docs/conf.py [hunk 4]
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
...
 
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, 'esrally', 'Rally Documentation',
-     [author], 1)
-]
+man_pages = [(master_doc, "esrally", "Rally Documentation", [author], 1)]
```
Comment (pquentin) [inline docs/conf.py:174]:
```suggestion
man_pages = [(master_doc, "esrally", "Rally Documentation", [author], 1,)]
```

File: docs/conf.py [hunk 5]
```diff
@@ -132,62 +126,57 @@ def read_min_es_version():
...
 # If true, show URL addresses after external links.
-#man_show_urls = False
+# man_show_urls = False
 
 # -- Options for Texinfo output -------------------------------------------
 texinfo_documents = [
-    (master_doc, 'Rally', 'Rally Documentation',
-     author, 'Rally', 'Macrobenchmarking framework for Elasticsearch.',
-     'Miscellaneous'),
+    (master_doc, "Rally", "Rally Documentation", author, "Rally", "Macrobenchmarking framework for Elasticsearch.", "Miscellaneous"),
```
Comment (pquentin) [inline docs/conf.py:181]:
```suggestion
    (master_doc, "Rally", "Rally Documentation", author, "Rally", "Macrobenchmarking framework for Elasticsearch.", "Miscellaneous",),
```

**Review by pquentin — COMMENTED** [review]
Thank you! Yes, your new proposal is the right thing to do: there is no good reason why those were left out, they were not linted so I did not format them.

I left a few trailing commas as suggestions, this will tell black to restore those lines as blocks.

**Review by pquentin — APPROVED** [review]
Thanks for bearing with me! LGTM.

**Pull request merged**